lib/Yam/Installation/Upgrade/PreviousRepoPage.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handle page that allows to select system/partition to be upgraded.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Installation::Upgrade::PreviousRepoPage;
```

Comment:
we don't need folders under `lib/Yam/` for now. Let's reorganize later and create all the pages at the same level.